hicommonwealth/commonwealth
# #5562 - Create Circle Buttons

This PR completes work detailed in issue #5562, which asks for the creation of a circular button containing only an icon (in primary and secondary variants). See design [here](https://www.figma.com/file/eIVp33a1oCu0AtcLwSbGjr/%F0%9F%9A%A7-Components-and-Patterns?type=design&node-id=3122-2051&mode=design&t=EEV66MAzwmhPjA9A-0)

## Link to Issue
Closes: #5562 

## Description of Changes
- Creates a new component called `CWCircleButton`
- Adds component to the component showcase page

## Test Plan
- visit `/components`
- navigate to section of page titled 'Circle Buttons' and ensure that they match look and behavior described in Figma above
